### PR TITLE
UI for Feature/traccar 2702 watch ext

### PR DESCRIPTION
**Pull request by amenk**
https://github.com/tananaev/traccar/issues/2702

### CODE REVIEW

File: web/l10n/de.json [hunk 1]
```diff
@@ -144,6 +144,8 @@
     "commandData": "Daten",
     "commandPhone": "Rufnummer",
     "commandMessage": "Nachricht",
+    "commandTimezoneOffset": "Zeitverschiebung zu UTC",
```
Comment (tananaev) [inline web/l10n/de.json]: You shouldn't change any of the localization files directly. All localization should be done via Transifex website.

Comment (amenk) [inline web/l10n/de.json]: okay ... also the english one? But in en.json it's fine, right? Shall I rebase or just revert the file?

Comment (tananaev) [inline web/l10n/de.json]: The way it works is that you only add/modify English strings, then Transifex automatically picks up the file directly from the repository and people can translate it. Then with every new release I update localization files using special script.

You can just revert it.

Comment (amenk) [inline web/l10n/de.json]: done

**Comment by tananaev** [comment]
If we rename "hearts" to "indicator", we should rename it here as well.

**Comment by amenk** [comment]
yep, done

File: web/l10n/en.json [hunk 2]
```diff
@@ -144,6 +144,8 @@
     "commandData": "Data",
     "commandPhone": "Phone Number",
     "commandMessage": "Message",
+    "commandTimezoneOffset": "Timezone Offset to UTC",
```
Comment (tananaev) [inline web/l10n/en.json]: Command key should match the command key for consistency. Also, I think we can shorten the value to "Timezone Offset".

File: web/l10n/en.json [hunk 3]
```diff
@@ -144,6 +144,8 @@
...
+    "commandSetIndicator": "Set Indicator",
```
Comment (tananaev) [inline web/l10n/en.json]: Please move this line next to other command names.

File: web/app/view/CommandDialog.js [hunk 4]
```diff
@@ -90,6 +90,33 @@ Ext.define('Traccar.view.CommandDialog', {
...
+            xtype: 'fieldcontainer',
+            reference: 'paramSetTimezone',
+            name: 'attributes',
+            hidden: true,
+
+            items: [{
+                xtype: 'numberfield',
+                fieldLabel: Strings.commandTimezoneOffset,
+                name: 'timezone',
+                minValue: -12,
```
Comment (tananaev) [inline web/app/view/CommandDialog.js]: I believe the value is in seconds. I guess you would have to change server side if it doesn't accept seconds.

Comment (amenk) [inline web/app/view/CommandDialog.js]: in the watch protocol it is in hours ...

Comment (tananaev) [inline web/app/view/CommandDialog.js]: API expects seconds, so it should be converted accordingly in the encoder.

File: web/l10n/en.json (hunk 3 above)
Comment (amenk) [inline web/l10n/en.json]: ok

File: web/l10n/en.json (hunk 2 above)
Comment (tananaev) [inline web/l10n/en.json]: The key in the command is "timezone", so it should be "commandTimezone".

**Comment by tananaev** [comment]
This is minutes as well, not seconds.

**Comment by tananaev** [comment]
Merged, thanks.

**Pull request merged**